PyOpenWorm/context.py

```diff
@@ -233,9 +222,17 @@ def rdf_object(self):
             # statement defined in some other context which specifically holds
             # context metadata.
             from PyOpenWorm.contextDataObject import ContextDataObject
-            self._rdf_object = ContextDataObject.contextualize(self.context)(ident=self.identifier)
+            self._rdf_object = ContextDataObject.contextualize(
+                self.context)(ident=self.identifier)
         return self._rdf_object
 
+    def __bool__(self):
+        return len(self._imported_contexts) != 0
```

Comment:
This should just `return True` - `__bool__`/`__nonzero__` is just needed so Python doesn't use `__len__` for the truth value.

Reply:
okok